builder/vmware/vmx/step_clone_vmx_test.go

```diff
@@ -23,6 +31,22 @@ func TestStepCloneVMX(t *testing.T) {
 	}
 	defer os.RemoveAll(td)
 
+	// Set up mock vmx file contents
+	var testCloneVMX = fmt.Sprintf("scsi0:0.filename = \"%s\"\n"+
```

Comment:
Will there always be a newline read into these items?

Reply:
Yes - the key/value pairs in the vmx file are always listed one per line. Is there a better way to achieve this?

Reply:
No, I think it's fine -- I just wanted to make sure that the test was accurate to the reality.